staker/block_validator.go

```diff
@@ -172,7 +172,12 @@ func (c *BlockValidatorConfig) Validate() error {
 }
 
 type BlockValidatorDangerousConfig struct {
-	ResetBlockValidation bool `koanf:"reset-block-validation"`
+	ResetBlockValidation bool               `koanf:"reset-block-validation"`
+	Revalidation         RevalidationConfig `koanf:"re-validation"`
```

Comment:
should just be `"revalidation"` without the dash.

Reply:
Done.